Cheerify/AddView.swift

```diff
@@ -85,6 +86,12 @@ struct AddView: View {
                 .foregroundColor(Color.white)
                 .background(Color(red: 196/255, green: 197/255, blue: 202/255))
                 .clipShape(RoundedRectangle(cornerRadius: 25))
+                .sheet(isPresented: $showPopUp) {
```

Comment:
this popup looks awesome! did you test it on device with a keyboard?

Reply:
thanks , I didn't test it then but now I have. 

will resolve the required changes

Reply:
also I am setting only light mode have used .preferredColorScheme(.light) as it seems there is no implementation for dark mode at this stage, but it can be added later if needed.